backend/backend/.env.example

```diff
@@ -0,0 +1,6 @@
+DEBUG=True
```

Comment:
@momanyisamuel, we still need to delete the one that's here, right?

Reply:
Yes this will be removed as discussed